miou/tests/dune

```diff
@@ -0,0 +1,12 @@
+(test
```

Comment:
Not sure that you want to execute this executable every time, it takes a very long time.

Reply:
how long is "very long"? And what are the options? marking it as executable? then we can't attach it to a package without a `public_name`...

Reply:
> how long is "very long"?

2mins on my (good) machine.

> And what are the options? marking it as executable? then we can't attach it to a package without a public_name...

We can set it as a simple executable if the test bother you too much (because it takes to much time).

Reply:
I'm fine leaving it as test. I usually develop in ocaml 4 land where I don't have miou anyways, and I prefer to have tests that are useful enabled and running (otherwise we hit the "what was the last time that feedback.exe worked" again).